### DISCUSSION TITLE
How to get the original expression string from *vm.Program?

### DISCUSSION THREAD

**Discussion by 100vlx** [antonmedv/expr#489] [General]
How to get the original expression string from *vm.Program?

The source field in *vm.Program is now lowercase...

**Comment by hesining** [comment]
I think it is necessary to provide a way to obtain the original expression, because ’Compile‘ and ’Run‘ may not be in the same place.Sometimes I need to know the original expression when executing 'Run'. It would be more convenient if  this method is provided.
